templates/breaking-changes-doc-template.md

```diff
@@ -0,0 +1,28 @@
+# Upgrade from v[PREVIOUS].x to v[CURRENT].x
```

Comment:
We've used this style of template variable elsewhere so I would suggest to do the same here

```suggestion
# Upgrade from v**PREVIOUS**.x to v**CURRENT**.x
```